llm/generate/gen_windows.ps1

```diff
@@ -243,6 +243,11 @@ if ($null -ne $script:CUDA_LIB_DIR) {
     init_vars
     $script:buildDir="../build/windows/${script:ARCH}/cuda$script:CUDA_VARIANT"
     $script:cmakeDefs += @("-A", "x64", "-DLLAMA_CUDA=ON", "-DLLAMA_AVX=on", "-DLLAMA_AVX2=off", "-DCUDAToolkit_INCLUDE_DIR=$script:CUDA_INCLUDE_DIR", "-DCMAKE_CUDA_ARCHITECTURES=${script:CMAKE_CUDA_ARCHITECTURES}")
+    if ($null -ne $script:OLLAMA_CUSTOM_CUDA_DEFS) {
+        write-host "OLLAMA_CUSTOM_CUDA_DEFS=`"${script:OLLAMA_CUSTOM_CUDA_DEFS}`""
+        $script:cmakeDefs +=@("${script:OLLAMA_CUSTOM_CUDA_DEFS}")
```

Comment:
Almost.  It should be `$env:OLLAMA_CUSTOM_CUDA_DEFS` scoped to grab this from the environment variables.

Reply:
Fixed in latest push. Apologies, I'm a bit weaker in Powershell and needed a refresher on env var references. I think it's gtg now.